src/DataCollectors/TraceDataCollector/Resources/Resources.resx

```diff
@@ -132,4 +132,7 @@
   <data name="VanguardConnectionTimeout" xml:space="preserve">
     <value>Failed to receive running event from CodeCoverage.exe in {0} seconds, This may occur due to machine slowness, please set environment variable {1} to increase timeout.</value>
   </data>
+  <data name="CodeCoverageOnlySupportsWindows" xml:space="preserve">
+    <value>Currently code coverage support available only for Windows operating system. No code coverage data available for current test run.</value>
```

Comment:
Get this verified by Pratap

Reply:
How about: "No code coverage data available. Code coverage is currently supported only on Windows."

Reply:
@pvlakshm sounds good. Thanks.